### DISCUSSION TITLE
markdown 导出

### DISCUSSION THREAD

**Discussion by Rainsheep** [halo-dev/halo#3233] [Q&A]
### Your current Halo version

2.2.0

### Describe this feature

首先想咨询一下，直接在 halo2 自带编辑器发布的文章是 html 文本吗，还是 markdown。

1. 如果是 html, 想了解一下从 halo1 的 md 转换为 halo2 的 html 有什么优点吗？
据我所知，对于单纯的博客系统而言，md 在书写和迁移过程中很方便。

2. 有计划支持导出文章为 markdown 的功能吗？ 或者仅能导出由 md 插件发布的文章？

**Comment by ruibaby** [comment]
1. 从 Halo 2 开始，我们的定位不会再是单纯的博客系统。
2. 默认编辑器是富文本编辑器，为了更多的内容类型。
3. 没有完全放弃 Markdown 编辑器，目前你可以安装编辑器插件，我们也有计划官方开发一个 Markdown 编辑器。
4. 导出功能期望由插件实现，但我认为不是目前紧急需要实现的功能。
